creator/data_templates/nodes.py

```diff
@@ -0,0 +1,29 @@
+from graphene import relay
+from graphql import GraphQLError
+from graphene_django import DjangoObjectType
+
+from creator.data_templates.models import DataTemplate
+
+
+class DataTemplateNode(DjangoObjectType):
+    class Meta:
+        model = DataTemplate
+        interfaces = (relay.Node,)
+        filter_fields = ()
```

Comment:
We might want to be able to filter by organization or study here at some point, right?

Reply:
Yea most likely. I still need to add in filtering